### DISCUSSION TITLE
[Desktop / Web] Allow searching in "Add to album" dialog

### DISCUSSION THREAD

**Discussion by zagatta** [ente-io/ente#3256] [Enhancements]
Since I have many albums and many pictures also in multiple albums, I would love to see the possiblity **to search** in the "Add to album" dialog. Right know (I think) I have to scroll to the right album, which can sometimes takes me 30+ seconds.

![signal-2024-09-13-132318](https://github.com/user-attachments/assets/2fa44208-52fd-4b85-a459-35fa2542a382)

It works on mobile, but it seems the possibility does not exist on Mac Desktop or Web.

**Comment by mnvr** [comment]
Not to take away from your feature request, but just so that you're aware - the latest release 1.7.5 (and web app) will now automatically add photos to the currently selected album when you drag and drop - which should cover the most important use case of quickly adding files to an album (though a search bar might be useful for other reasons)
